### ISSUE TITLE
Icon view should folow List View sorting

### ISSUE THREAD

**Issue by bwakkie** [linuxmint/nemo#2848]
Nemo 5.0.3

When I change the sort in list view I expect the same sort in icon view.

Upon opening a folder in Icon view add the ability to sort in this view.
I suggest to add the column names in the top of the icon view as buttons indicating which way they sort.

**Comment by Jeremy7701** [comment]
There is no sort selection (as such) in list view.
You can select sort by some ascending/descending column.

In icon view you select sort by some named option (including 'manual' - haven't tried that one yet).

**Comment by bwakkie** [comment]
Yes I see now I can indeed order the same way as in the list view. Thanks for pointing this out. This functionality is 'hidden' too deep in my opinion. Now you have to right-click somewhere on a white space between the file icons to get to the "Arrange items" function. Would it not be way clearer to keep the list view 'headers' in the top as a visual representation of these order options?
The manual option is an interesting function I haven't seen before in file managers as you can drag files into 'clusters' even on top of each other and after restarting Nemo it is remembered. It is also remembered of you change the order and move back to 'manual' order.
